processing/SpaceCrashAlpha/Projectile.pde

```diff
@@ -7,38 +7,36 @@ abstract class Projectile extends DisplayableBase {
   float velocityMagnitude = 10;
   float brightness = 255;
   int framesLeft = 100;
-  PVector position;
-  PVector lastPosition;
-  float angle;
-  PVector velocity;
   Being owner;
+  PointMass physicsModel;
   
   Projectile(Being owner, PVector position) {
-    this.position = position.copy();
-    lastPosition = this.position.copy();
-    angle = owner.angle;
-    velocity = PVector.fromAngle(angle);
-    velocity.mult(velocityMagnitude);
+    physicsModel = new PointMass(false);
+    physicsModel.setDrag(0.0);
+    physicsModel.setPosition(position.copy());
+    physicsModel.setLastPosition(position.copy());
+    float angle = owner.angle;
+    PVector velocity = PVector.fromAngle(angle).mult(velocityMagnitude);
+    physicsModel.setVelocity(velocity);
+    
+    physicsObjects.add(physicsModel);
```

Comment:
doing this based on the way we're adding projectiles to the `projectiles` collection in `Player.fire()`. but does this cause a memory leak? also, how is `physicsObjects` in scope here?